target/linux/pistachio/dts/pistachio/pistachio_marduk_common.dtsi

```diff
@@ -17,6 +17,7 @@
 		serial0 = &uart0;
 		serial1 = &uart1;
 		ethernet0 = &enet;
+		wifi0 = &wifi;
```

Comment:
In that case, ethernet0 would be but eth0 but it is eth0 and there is no convention for wifi alias.